code/datums/movement/mob.dm

```diff
@@ -113,7 +113,7 @@
 	if(mob.buckled) // Wheelchair driving!
 		if(isspaceturf(mob.loc))
 			return // No wheelchair driving in space
-		if(istype(mob.buckled, /obj/structure/bed/chair/wheelchair))
+		if(istype(mob.buckled, /obj/structure/bed/chair/wheelchair) || istype(mob.buckled, /obj/structure/bed/chair/janicart))
```

Comment:
I get it's not a major part of this PR but yuk.

Reply:
This is temporary at least, when the other movement tweaks PR goes in I can resolve this to use an override instead.

Reply:
Rebasing to remove it now.